examples/evaluation/README.md

```diff
@@ -24,4 +28,6 @@ Run in this directory with Python.
 python evaluate_on_binary_classifier.py
 python evaluate_on_multiclass_classifier.py
 python evaluate_on_regressor.py
+python evaluate_with_custom_metrics.py
+python evaluate_with_custom_metrics_comprehensive.py
```

Comment:
While we're at it, can we add these to our CI? https://github.com/mlflow/mlflow/blob/5784e7e833385e59cd194fd63e4ae5e456abd779/tests/examples/test_examples.py#L138-L191